### PR TITLE
Api split

### PR DESCRIPTION
**Pull request by FantasticoFox**
splits the API class into two files:
APIRead (Standard path)
and APIWrite (write path).

To set specific permissions.

Fixed also container names in tools

### CODE REVIEW

File: includes/APIRead.php [hunk 1]
```diff
@@ -83,18 +83,16 @@ function updateDomainManifest($witness_event_id, $db) {
...
-        'verify_page', 
-        'get_page_by_rev_id', 
-        'page_all_rev', 
-        'get_page_last_rev', 
-        'get_witness_data', 
-        'request_merkle_proof', 
-        'store_signed_tx', 
-        'store_witness_tx', 
-        'request_hash' 
+        'verify_page', //READ
```
Comment (rht) [inline includes/APIRead.php:89]: We already know that the list has all read-only API endpoints. So the `//READ` comments can be removed.

File: includes/APIWrite.php [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+        $conds,
+    );
+    foreach ($res as $row) {
+        array_push($out, $row->{$col});
+    }
+    return $out;
+}
+
+// TODO move to Util.php
+function updateDomainManifest($witness_event_id, $db) {
```
Comment (rht) [inline includes/APIWrite.php:34]: This probably needs to exist only on APIWrite. You need to remove the identical function in APIRead.

File: includes/APIWrite.php [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+use WikiPage;
+
+# include / exclude for debugging
+error_reporting(E_ALL);
+ini_set("display_errors", 1);
+
+require_once("ApiUtil.php");
+require_once("Util.php");
+
+function selectToArray($db, $table, $col, $conds) {
```
Comment (rht) [inline includes/APIWrite.php:20]: Is this used in APIWrite and APIRead, or only here?

File: includes/APIRead.php (hunk 1 above)
Comment (FantasticoFox) [reply includes/APIRead.php:89]: Did I delete too much by removing line 97? I need to check that.

File: includes/APIWrite.php (hunk 2 above)
Comment (FantasticoFox) [reply includes/APIWrite.php:34]: Yes, makes sense.

File: includes/APIWrite.php (hunk 3 above)
Comment (FantasticoFox) [reply includes/APIWrite.php:20]: Only in write only. Needs to be removed in read.

Same true for domain manifest.

**Comment by FantasticoFox** [comment]
Will resubmit to dev

File: includes/APIRead.php (hunk 1 above)
Comment (FantasticoFox) [reply includes/APIRead.php:89]: Done [API_Split e95bf1e] Remove Read comments

File: includes/APIWrite.php (hunk 2 above)
Comment (FantasticoFox) [reply includes/APIWrite.php:34]: Done - [API_Split bb2b631] Remove unnecessary function updateDomainManifest from APIRead

File: includes/APIWrite.php (hunk 3 above)
Comment (FantasticoFox) [reply includes/APIWrite.php:20]: Open. Let's check together!

**Pull request closed without merging**